control_plane/src/bin/neon_local.rs

```diff
@@ -49,6 +49,7 @@ use utils::{
 const DEFAULT_SAFEKEEPER_ID: NodeId = NodeId(1);
 const DEFAULT_PAGESERVER_ID: NodeId = NodeId(1);
 const DEFAULT_BRANCH_NAME: &str = "main";
+const DEFAULT_RETRY_UNTIL_SECS: u64 = 10;
```

Comment:
WHy is this needed? We have the `default_value("10")` for the CLI flag.

Reply:
removed it